Python/README.md

```diff
@@ -1,38 +1,52 @@
 # Python
 
-macOS, like Linux, ships with [Python](http://python.org/) already installed. But you don't want to mess with the system Python (some system tools rely on it, etc.), so we'll install our own version(s). There are two ways to install Python, (1) Homebrew and (2) Pyenv. If you plan to use multiple versions of Python (e.g. 2, 3, and anaconda) then you should use pyenv.
+macOS, like Linux, ships with [Python](https://python.org/) already installed.
+But you don't want to mess with the system Python (some system tools rely on
+it, etc.), so we'll install our own version(s). There are two ways to install
+Python, (1) Homebrew and (2) Pyenv. If you plan to use multiple versions of
+Python (e.g. 2, 3, and anaconda) then you should use pyenv.
+
+## Installation
 
 ### Homebrew method
 
-The following command will install Python 2.7 and any dependencies required (it can take a few minutes to build everything):
+The following command will install Python 2.7 and any dependencies required:
```

Comment:
`brew install python` now install Python 3:

https://docs.brew.sh/Homebrew-and-Python

Reply:
Odd, what I can find it says:
>The executables are organized as follows so that Python 2 and Python 3 can both be installed without conflict:
>- `python3` points to Homebrew’s Python 3.x (if installed)
>- `python2` points to Homebrew’s Python 2.7.x (if installed)
>- `python` points to Homebrew’s Python 2.7.x (if installed) otherwise the macOS system Python. Check out `brew info python` if you wish to add Homebrew’s 3.x `python` to your `PATH`.

From this I understand that running `brew install python` installs version 2.7, and `python3` installs version 3 etc. But when I run `brew info python` I get information about version 3.

Reply:
I have both versions on my machine and I can't remember how I installed them.. can anyone verify anything related to this?

Reply:
They changed it fairly recently, so how you got them may differ from now.

> Homebrew provides one formula for Python 3.x (`python`) and another for Python 2.7.x (`python@2`).

Note `python` and `python@2` are the Brew formula names. The bit you quoted are the installed executable names.

If you uninstall all and re-install fresh, you'll get Python 3 with `brew install python` and Python 2 with `brew install python@2`.

Reply:
Cool, I'll change this bit and then I'll merge, considering you approved it.